### PR TITLE
[13.0][MIG] crm_lead_code

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Sequential Code for Leads / Opportunities
================================
This module adds a sequential code for leads / opportunities.

### CODE REVIEW

File: crm_lead_code/__manifest__.py
Comment (JayVora-SerpentCS) [inline crm_lead_code/__manifest__.py]: Website : https://github.com/OCA/crm

Comment (JayVora-SerpentCS) [inline crm_lead_code/__manifest__.py]: https://github.com/OCA/crm

File: crm_lead_code/models/crm_lead.py
Comment (JayVora-SerpentCS) [inline crm_lead_code/models/crm_lead.py]: Remove all the decorators @api.multi, @api.returns, @api.one from the code, as now they are all multi-record by default. In case of the last one, you will need to adapt the code to the behavior change.

File: crm_lead_code/__manifest__.py
Comment (Swapnesh-SerpentCS) [inline crm_lead_code/__manifest__.py]: data should be first

File: crm_lead_code/models/crm_lead.py
Comment (Swapnesh-SerpentCS) [inline crm_lead_code/models/crm_lead.py]: IMO we should use `next_by_id` here

File: crm_lead_code/models/crm_lead.py
Comment (Swapnesh-SerpentCS) [inline crm_lead_code/models/crm_lead.py]: Instead of assigning  `default['code'] ` cant we add `copy=False` on field. (Which will automatically update code on copy )

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by HaraldPanten** [comment]
Hi @murtuzasaleh ,

Could you check travis, please?

**Review by HaraldPanten — APPROVED** [review]
Functional review. LGTM 👍

File: crm_lead_code/__manifest__.py
Comment (skeller1) [inline crm_lead_code/__manifest__.py:26]: Please fix the pre-commit error with `"`

File: crm_lead_code/static/description/index.html
Comment (skeller1) [inline crm_lead_code/static/description/index.html:6]: Please (re)generate the docs

**Review by skeller1 — CHANGES_REQUESTED** [review]

**Review by mamcode — APPROVED** [review]
LGTM.

**Comment by mamcode** [comment]
This PR has enough approvals, it should be merged soon... @OCA/crm-sales-marketing-maintainers

**Review by Swapnesh-SerpentCS — APPROVED** [review]

**Pull request merged**
